docs/UserCommands.md

```diff
@@ -0,0 +1,49 @@
+# User Command Reference
+
+`@rocket user`
+
+## Options to specify input
+
+* `edit`
+  * `--name` yourname
+  * `--email` emailaddress
+  * `--pos` yourposition
+  * `--github` yourrepo
+  * `--major` yourmajor
+  * `--bio` yourbio
+    * edit properties of your Launch Pad profile
+  * ADMIN only option: `--member` MEMBER_ID
+    * edit properties of another user's Launch Pad profile
+
+* `view` MEMBER_ID
+  * view information about a user
+
+* `help`
+  * outputs options for user commands
+
+## ADMIN only
+
+* `delete` MEMBER_ID
+  * permanently delete member's Launch Pad profile
+
+* `add_admin` MEMBER_ID
```

Comment:
It might make sense to add a dedicated `admin` or `permissions` subcommand that will subsume both `add_admin` and `remove_admin` (and maybe the team lead commands as well? That may make handling permissions more difficult later on though).